packages/router/router.ts

```diff
@@ -2279,7 +2300,11 @@ function normalizeNavigateOptions(
   }
 
   // Create a Submission on non-GET navigations
-  if (opts.formMethod != null && opts.formMethod !== "get") {
+  if (
+    opts.formMethod != null &&
+    opts.formMethod !== "get" &&
+    opts.formMethod !== "head"
```

Comment:
Made sure we're always including get/head methods together

Reply:
should `options` be considered too here?

Reply:
For `OPTIONS` we throw during SSR requests, but it looks like we don't have that check in the client-side flows - probably an oversight since it's not a valid TS method.  I changed my mind on adding `HEAD` to the client side flows as well (Remix doesn't support that today either), so am pushing up some changes to strengthen the method checks across the board.  Both `HEAD` and `OPTIONS` will throw errors on client side navigations/fetches, and only `OPTIONS` will throw an error on SSR requests